src/pages/vans.jsx

```diff
@@ -1,36 +1,27 @@
-import { useEffect } from "react";
-import { useState } from "react";
-import Footer from "../components/Footer";
-import { Link } from "react-router-dom";
 import { useSelector } from "react-redux";
 import { useDispatch } from "react-redux";
 import { fetching } from "../features/vanslice";
+import { useLocation, Link, useSearchParams } from "react-router-dom";
 import "../server";
+import { getVans } from "../api";
+import { useLoaderData } from "react-router-dom";
+
+export function loader(){
+  return getVans()
+}
 
 export default function Vans() {
-  const [Vans, setVans] = useState([]);
-  const [loads, setLoads] = useState(true);
+  const Vans = useLoaderData();
+  const location = useLocation();
+  const loads=false;
```

Comment:
Remove this variable, as it is not being used anywhere in the component.

Reply:
DONE :heavy_check_mark: